### ISSUE TITLE
Autobuild .deb package

### ISSUE THREAD

**Issue by lunemec** [lunemec/nanny#18]
Since we are autobuilding Docker, we should also autobuild .deb package, but how?

**Comment by PhilipSchmid** [comment]
Yeah why not, I think this would be helpful. But to be honest, currently I don't know how to build a .deb package. How have you built the ones which you have already released? Perhaps we could use Travis-CI to do this in an automated way?

Regards,
Philip

**Comment by lunemec** [comment]
At work, we use gitlab with CI/CD and use that to build .deb packages. The ones that are in the release sections were build manually within debian docker image. Basically I have prepared docker debian container with the build scripts and used that locally. 

I'm not sure if travis can do this kind of thing?
